src/test/kotlin/org/rust/ide/intentions/MatchToIfLetIntentionTest.kt

```diff
@@ -80,13 +59,44 @@ class MatchToIfLetIntentionTest : RsIntentionTestBase(MatchToIfLetIntention::cla
         fn main() {
             let color = OptionColor::Color(255, 255, 255);
 
-            if let _ = color { print!("No color") };
+            if let OptionColor::Color(255, 255, 255) = color { 1 } else if let OptionColor::Color(_, _, _) = color { 2 } else if let OptionColor::NoColor = color { 3 };
```

Comment:
```suggestion
    if let OptionColor::Color(255, 255, 255) = color {
        1
    } else if let OptionColor::Color(_, _, _) = color {
        2
    } else if let OptionColor::NoColor = color { 
        3
    };
```

I think we should make if blocks multiline

Reply:
I wonder how to do this cleanly. It is currently done for normal expressions, but if the input expression was already a block expression, it has braces, so we need to somehow take the contents inside the braces out of it. I did it in a very crude way, I hope that it's ok, since match arm braces usually don't contain things like `move`, `unsafe` or attributes on the block.

Reply:
I think your approach is fine